src/main/java/minegame159/meteorclient/modules/movement/Jesus.java

```diff
@@ -115,8 +115,8 @@
     );
 
     private final Setting<Boolean> dipIntoLavaIfFireResistance = sgLava.add(new BoolSetting.Builder()
-            .name("dip-into-lava-if-fire-resistance")
-            .description("Lets you go under the lava if you have Fire Resistance effect to avoid fall damage.") // rofl someone put "fall damage" here
+            .name("dip-if-fire-resistance")
+            .description("Lets you go under the lava if you have Fire Resistance effect to avoid fall damage.") // rofl some retard put "fall damage" here
```

Comment:
Lava doesn't stop you taking fall damage

Reply:
That was not me that wrote that but I'll change it yes.